package.json

```diff
@@ -130,6 +130,20 @@
     "workbox-webpack-plugin": "^6.5.4",
     "yaml-loader": "^0.8.0"
   },
+  "overrides": {
+    "appium-safari-driver": {
+      "archiver-utils": {
+        "glob": {
+          "foreground-child": {
+            "cross-spawn": "^7.0.5"
```

Comment:
From a quick search, I think it should just be `appium-safari-driver -> cross-spawn` and remove the archiver-utils -> glob -> foreground-child. Which would align with `node_modules/appium-safari-driver/node_modules/cross-spawn`

Reply:
Yes, I tried that also but it did not work

Reply:
This is the used library between all packages:
```bash
/cf-identity-wallet
├─┬ @capacitor/assets@3.0.5
│ └─┬ @trapezedev/project@7.1.3
│   └── cross-spawn@7.0.6 deduped
├─┬ @capacitor/cli@6.2.0
│ └─┬ @ionic/utils-subprocess@2.1.11
│   └── cross-spawn@7.0.6 deduped
├─┬ @wdio/cli@9.4.1
│ └─┬ execa@9.5.2
│   └── cross-spawn@7.0.6 deduped
├─┬ @wdio/cucumber-framework@9.3.1
│ └─┬ glob@10.4.5
│   └─┬ foreground-child@3.3.0
│     └── cross-spawn@7.0.6 deduped
├─┬ appium-chromium-driver@1.4.5
│ └─┬ appium-chromedriver@6.1.7
│   └─┬ @appium/support@5.1.8
│     └─┬ glob@10.4.5
│       └─┬ foreground-child@3.3.0
│         └── cross-spawn@7.0.6
├─┬ appium-safari-driver@3.5.18 overridden
│ └─┬ archiver-utils@5.0.2 extraneous overridden
│   └─┬ glob@10.4.5
│     └─┬ foreground-child@3.3.0 overridden
│       └── cross-spawn@7.0.3
├─┬ appium-uiautomator2-driver@3.9.2
│ ├─┬ appium-adb@12.7.2
│ │ └─┬ @appium/support@6.0.0
│ │   └─┬ glob@10.4.5
│ │     └─┬ foreground-child@3.3.0
│ │       └── cross-spawn@7.0.6 deduped
│ └─┬ appium@2.13.1
│   └─┬ cross-env@7.0.3
│     └── cross-spawn@7.0.6
├─┬ appium-xcuitest-driver@7.33.0
│ └─┬ appium-remote-debugger@12.1.4
│   └─┬ glob@10.4.5
│     └─┬ foreground-child@3.3.0
│       └── cross-spawn@7.0.6
├─┬ cross-env@7.0.3
│ └── cross-spawn@7.0.6
├─┬ eslint@8.57.1
│ └── cross-spawn@7.0.6 deduped
├─┬ jest@29.7.0
│ └─┬ @jest/core@29.7.0
│   └─┬ jest-changed-files@29.7.0
│     └─┬ execa@5.1.1
│       └── cross-spawn@7.0.6 deduped
├─┬ lint-staged@15.2.11
│ └─┬ execa@8.0.1
│   └── cross-spawn@7.0.6 deduped
├─┬ webpack-cli@5.1.4
│ └── cross-spawn@7.0.6 deduped
└─┬ webpack-dev-server@4.15.2
  └─┬ default-gateway@6.0.3
    └─┬ execa@5.1.1
      └── cross-spawn@7.0.6 deduped
```

The only dependency affected is:

```bash
├─┬ appium-safari-driver@3.5.18 overridden
│ └─┬ archiver-utils@5.0.2 extraneous overridden
│   └─┬ glob@10.4.5
│     └─┬ foreground-child@3.3.0 overridden
│       └── cross-spawn@7.0.3
```

Reply:
OK but the other packages look ok - have you tried on a fresh install after wiping node_modules? https://github.com/npm/cli/issues/5850

Reply:
Yes, I tried but it did not work